### PR TITLE
Update Crop_api && ImageWidgetCrop doc.

### PR DESCRIPTION
**Pull request by woprrr**

### CODE REVIEW

**Comment by woprrr** [comment]
@slashrsm All is ok for u with that changes ?

File: modules/image_widget_crop/intro.md [hunk 1]
```diff
@@ -1,30 +1,60 @@
 # ImageWidgetCrop module
 
-[ImageWidgetCrop](https://www.drupal.org/sandbox/woprrr/2571403) provides an interface for using the features of the [Crop API](https://www.drupal.org/project/crop). This widget provide an UX for use a crop on all fields images. This module have particularity to purpose capability to crop the same image by 'Crop type' configured. It's very usefull for editorial sites or media management sites.
+[ImageWidgetCrop](https://www.drupal.org/project/image_widget_crop) provides an interface for using the features of the [Crop API](https://www.drupal.org/project/crop). This widget provide an UX for use a crop on all fields images. This module have particularity to purpose capability to crop the same image by 'Crop type' configured. It's very usefull for editorial sites or media management sites.
 
-## Try me
+## Try demo module
+You can Test ImageWidgetCrop in action directly with the sub-module "ImageWidgetCrop example" to test differents usecase of this module.
```
Comment (slashrsm) [inline modules/image_widget_crop/intro.md:6]: different use cases

File: modules/image_widget_crop/intro.md [hunk 2]
```diff
@@ -1,30 +1,60 @@
...
 
-You can Test ImageWidgetCrop in action directly with the sub-module, "ImageWidgetCrop example" to test different usecases of this module.
+You can try the module live with the website dedicated to demonstrate the module [DEMO SITE](http://www.image-widget-crop.com) with demo user
```
Comment (slashrsm) [inline modules/image_widget_crop/intro.md:8]: You can also try the module on the demo site.

No need to use caps.

File: modules/image_widget_crop/intro.md [hunk 3]
```diff
@@ -1,30 +1,60 @@
...
+[ImageWidgetCrop](https://www.drupal.org/project/image_widget_crop) provides an interface for using the features of the [Crop API](https://www.drupal.org/project/crop). This widget provide an UX for use a crop on all fields images. This module have particularity to purpose capability to crop the same image by 'Crop type' configured. It's very usefull for editorial sites or media management sites.
 
-## Try me
+## Try demo module
+You can Test ImageWidgetCrop in action directly with the sub-module "ImageWidgetCrop example" to test differents usecase of this module.
 
-You can Test ImageWidgetCrop in action directly with the sub-module, "ImageWidgetCrop example" to test different usecases of this module.
+You can try the module live with the website dedicated to demonstrate the module [DEMO SITE](http://www.image-widget-crop.com) with demo user 
+Psw: "demo"
```
Comment (slashrsm) [inline modules/image_widget_crop/intro.md:9]: Password

File: modules/image_widget_crop/intro.md [hunk 4]
```diff
@@ -1,30 +1,60 @@
...
+User: "demo"
+
+You can also see [Demo vidéo](https://www.youtube.com/watch?v=Ioo_bYnCOOg)
```
Comment (slashrsm) [inline modules/image_widget_crop/intro.md:12]: You can also see the demo video.

File: modules/image_widget_crop/intro.md [hunk 5]
```diff
@@ -1,30 +1,60 @@
...
 
 ## Requirements
 
 1. [Crop API](https://www.drupal.org/project/crop)
+1. [Cropper Library](https://github.com/fengyuanchen/cropper)
+
+##Installation
+
+1. Download and extract the module to your (`sites/all/modules/contrib`) folder.
```
Comment (slashrsm) [inline modules/image_widget_crop/intro.md:21]: I'd rather say "Download the module using standard install process." People might have different directory structure for example.

File: modules/image_widget_crop/intro.md [hunk 6]
```diff
@@ -1,30 +1,60 @@
...
+2. Enable the module on the Drupal Modules page (`admin/modules`) or using
+   $ drush en
+
+The module is currently using Cropper as a library to display,
+ the cropping widget.
+To properly configure it, do the following:
+
+* Local library:
+  1. Download the latest version of Cropper at
```
Comment (slashrsm) [inline modules/image_widget_crop/intro.md:30]: s/at/from

File: modules/image_widget_crop/intro.md [hunk 7]
```diff
@@ -36,27 +66,37 @@ You can Test ImageWidgetCrop in action directly with the sub-module, "ImageWidge
...
-  ![Step 16](images/step_16.png)
-  ![Step 17](images/step_17.png)
-  
-## ImageWidgetCrop Code Quality
-
-* [SensioLabsInsight](https://insight.sensiolabs.com/projects/0e2f44af-6837-4772-b3e0-c373faa95ae6)
-* [Scrutinizer](https://scrutinizer-ci.com/g/woprrr/image_widget_crop/?branch=8.x-1.x)
-* [Travis-ci](https://travis-ci.org/woprrr/image_widget_crop)
+3. General configuration
+  * ImageWidgetCrop module can be implemented in few usecases. Except with Fields usage the configuration of (`image_crop`) element in (`admin/config/media/crop-widget`).
```
Comment (slashrsm) [inline modules/image_widget_crop/intro.md:70]: This sounds strange.

File: modules/image_widget_crop/intro.md [hunk 8]
```diff
@@ -36,27 +66,37 @@ You can Test ImageWidgetCrop in action directly with the sub-module, "ImageWidge
...
+    ![General configuration](images/step_18.png)
+
+### FormApi
+If we need to implement `image_crop` element manually to your forms you can use this general configuration or set all options manualy in code.
```
Comment (slashrsm) [inline modules/image_widget_crop/intro.md:74]: This too.

**Pull request closed without merging**